src/app/InteractionModelEngine.h

```diff
@@ -91,8 +108,13 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate
 
         struct
         {
-            bool mShouldContinueProcessing; ///< Set to true if update is allowed.
+            bool mShouldContinueProcessing; ///< Set to true if invoke command is allowed.
         } mIncomingInvokeCommandRequest;
+
+        struct
+        {
+            bool mShouldContinueProcessing; ///< Set to true if read requestt is allowed.
```

Comment:
s/request/request.

A better name seems to be `readRequestAllowed`  (and above it would be `invokeAllowed`)

Reply:
Done, thanks